tests/python/test_bit_array_vectorization.py

```diff
@@ -41,3 +41,49 @@ def verify():
     init()
     assign_vectorized()
     verify()
+
+
+@ti.test(require=ti.extension.quant, debug=True, cfg_optimization=False)
+def test_offset_load():
+    ci1 = ti.type_factory_.get_custom_int_type(1, False)
```

Comment:
```suggestion
    ci1 = ti.type_factory.custom_int(1, False)
```
Also the identifier should be `cu1`?